features/teacher.feature

```diff
@@ -0,0 +1,44 @@
+Feature: teacher login functionality
+
+    As a teacher who has an application
+    So that I can edit my information
+    I can login via gmail
+
+Scenario: Logging in as a teacher should be able to edit their info
+  Given the following schools exist:
+  |       name      |     city     |  state  |            website            |
+  |   UC Berkeley   |   Berkeley   |   CA    |   https://www.berkeley.edu    |
+  Given the following teachers exist:
+  | first_name | last_name | admin | email                | school_id |
+  | Joseph     | Mamoa     | false | testteacher@berkeley.edu | 1 |
+  Given I have a teacher email
+  Given I am on the BJC home page
+  And   I follow "Log In"
+  Then  I can log in with Google
+  And   I see a confirmation "You can edit your information"
+  Then  the "First Name" field should contain "Joseph"
+  And   I enter my "First Name" as "Joe"
+  And   I enter my "School Name" as "Cupertino High School"
+  And   I enter my "City" as "Cupertino"
+  And   I select "CA" from "State"
```

Comment:
I think you need to select education level before submitting.

Reply:
you are right!